### PR TITLE
Fix nearly all bugs noticed on test

### PR DESCRIPTION
**Pull request by denisVorobyovExceed**

### CODE REVIEW

File: src/appointment/NewTab/NewTab.js [hunk 1]
```diff
@@ -7,6 +7,9 @@ import {
   TextField,
   MenuItem
 } from '@material-ui/core';
+// import AdapterDateFns from '@material-ui/lab/AdapterDateFns';
```
Comment (usatovaDasha) [inline src/appointment/NewTab/NewTab.js:10]: remove this rows

File: src/appointment/NewTab/NewTab.js [hunk 2]
```diff
@@ -102,6 +105,13 @@ function NewTab({
               }}
               onChange={e => setDate(e.target.value)}
             />
+            {/* <LocalizationProvider dateAdapter={AdapterDateFns}>
```
Comment (usatovaDasha) [inline src/appointment/NewTab/NewTab.js:108]: remove this rows

File: src/appointment/NewTab/NewTab.scss [hunk 3]
```diff
@@ -22,7 +22,7 @@
   justify-content: space-between;
 
   .new-tab-row { 
-    width: 100%;
+    width: calc(100% / 3);
```
Comment (usatovaDasha) [inline src/appointment/NewTab/NewTab.scss:25]: 33% ?

File: src/login/LoginPage/LogInPage.scss [hunk 4]
```diff
@@ -27,7 +27,23 @@
...
+  }
+
+  .loginpage-right {
+    width: 60vw;
+    text-align: center;
+  }
+}
+
+@media (max-width: 768px) {
+
```
Comment (usatovaDasha) [inline src/login/LoginPage/LogInPage.scss:41]: remove this row

File: src/appointment/NewTab/NewTab.scss (hunk 3 above)
Comment (denisVorobyovExceed) [reply src/appointment/NewTab/NewTab.scss:25]: 33,33333333333333333...

File: src/appointment/Appointment/Appointment.js [hunk 5]
```diff
@@ -125,52 +132,56 @@ export default function Appointment({ title, setTitle }) {
   }
 
   const handleHide = () => {
-    if(!hide.status) setHide({status: true, text: "Добавить прием"});
-    else setHide({status: false, text: "Скрыть"});
+    if(!hideNewTab) setHideNewTab(true);
```
Comment (usatovaDasha) [inline src/appointment/Appointment/Appointment.js:135]: setHideNewTab(!hideNewTab)

File: src/appointment/Appointment/Appointment.js [hunk 6]
```diff
@@ -125,52 +132,56 @@ export default function Appointment({ title, setTitle }) {
...
-        return "scroll-table-filter-on-hide";
-      case Boolean(!filterOn && hide.status): 
-        return "scroll-table-hide";
-      case Boolean(filterOn && !hide.status): 
-        return "scroll-table-filter-on";
-      case Boolean(!filterOn && !hide.status): 
-        return "scroll-table";
-    }
-  }
+  // const classTable = () => {
```
Comment (usatovaDasha) [inline src/appointment/Appointment/Appointment.js:149]: remove

File: src/appointment/Appointment/Appointment.js [hunk 7]
```diff
@@ -125,52 +132,56 @@ export default function Appointment({ title, setTitle }) {
...
-        />}
-        {!filterSortHide
+            doctors={doctors} 
+            getAppointments={getAppointments} 
+            handleError={handleError}
+            handleHide={handleHide}
+            hideNewTab={hideNewTab}
+          />
+        }
+        {!hideSortAndFilter
```
Comment (usatovaDasha) [inline src/appointment/Appointment/Appointment.js:184]: !hideSortAndFilter && filterOn && < FilterBar

File: src/appointment/Appointment/Appointment.scss [hunk 8]
```diff
@@ -5,17 +5,9 @@
...
-  overflow-y: auto;
-  overflow-x: hidden;
-}
-
-.scroll-table-filter-on {
-  margin-top: 12px;
-  height: 49vh; 
   width: 100vw;
   overflow-y: auto;
+  overflow-x: autp;
```
Comment (usatovaDasha) [inline src/appointment/Appointment/Appointment.scss:10]: ?????

File: src/appointment/Appointment/Appointment.scss [hunk 9]
```diff
@@ -42,18 +34,23 @@
...
+.anchor {
+  width: 1px;
+  height: 1px;
+}
+
 @media (max-width: 1024px) {
-  .show-button {
-    display: none;
-  }
+  // .show-button {
```
Comment (usatovaDasha) [inline src/appointment/Appointment/Appointment.scss:43]: remove

File: src/appointment/List/List.scss [hunk 10]
```diff
@@ -1,8 +1,14 @@
...
+  width: 100%;
+  padding: 0;
+}
+
 .table-container.MuiTableContainer-root {
   background: white;
   border-spacing: 0px;
   border-collapse: collapse;
-  overflow-x: hidden;
+  overflow-x: scroll;
```
Comment (usatovaDasha) [inline src/appointment/List/List.scss:10]: overflow: scroll;

File: src/login/InputLogin/InputLogin.js [hunk 11]
```diff
@@ -34,7 +34,7 @@ function InputLogin({ setInputParent, isLoginOccupied }) {
   return (
     <TextField 
       variant="outlined"
-      size="small" 
+      // size="small"
```
Comment (usatovaDasha) [inline src/login/InputLogin/InputLogin.js:37]: remove

File: src/login/InputPassRepeat/InputPassRepeat.js [hunk 12]
```diff
@@ -16,17 +23,38 @@ function InputPassRepeat({ setInputParent, firstPass }) {
...
-        setInputParent(e.target.value);
-        setPass(e.target.value);
-        comparePass(e.target.value);
-      }}
+      onChange={e => handleChange(e)}
+      endAdornment={
+        <InputAdornment position="end">
+          <IconButton
+            aria-label="toggle password visibility"
+            onClick={handleClickShowPassword}
```
Comment (usatovaDasha) [inline src/login/InputPassRepeat/InputPassRepeat.js:50]: correct call function

**Pull request merged**